mkdocs.yml

```diff
@@ -42,6 +42,8 @@ nav:
       - Pipeline specific instructions:
         - fmriprep: "nipoppy/workflow/proc_pipe/fmriprep.md"
         - mriqc: "nipoppy/workflow/proc_pipe/mriqc.md"
+  - Contributing:
+    - Pull requests: "contributing/pull_requests.md"
```

Comment:
Is there something we can combine with this page so the navbar on the left doesn't look so lonely with just the `Pull requests` section?

Reply:
I see what you mean. I’m not entirely sure what existing section would also fall under “Contributing” TBH - do you have something in mind? I saw this tab also as a placeholder that we can add to with other process-related things in the future.

One option would be to put the separate page we originally had describing “Our team” (https://github.com/neurobagel/documentation/blob/main/docs/team.md - not currently visible in the nav) under this tab, so we can also shorten the home page a bit. Does that sound good?

Reply:
This has been implemented!